news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
+
+The set also has nicely done lower diffs, including his signature "Basic" difficulty, which introduces you into the most basic structures of osu!taiko's dons and kats (the red and blue notes!).
 
-[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/proof-of-existance.jpg)](https://osu.ppy.sh/s/427864)
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/proof-of-existence.jpg)](https://osu.ppy.sh/s/427864)
 
-What a high quality set brought up by tasuke912 and SKSalt this month ! What made this chart get featured in the Spotlights is the management of a flawless structure, patterning and flow in each of their respective difficulties, really making a contrast with some other ranked maps this month. tasuke912's pair patterns usage and rhythm changes in the Inner Oni is the signature of a very highly experienced beatmapper, representing the song extremely accurately while making it fun to play through flow and hitsounding.
+What a high quality set brought up by **[tasuke912](https://osu.ppy.sh/users/2774767)** and **[SKSalt](https://osu.ppy.sh/users/3280542)** this month! What made this chart get featured in the Spotlights is the management of a flawless structure, patterning and flow in each of their respective difficulties, really contrasting with some other ranked maps this month. tasuke912's pair patterns usage and rhythm changes in the Inner Oni is the signature of a very highly experienced beatmapper, representing the song extremely accurately while making it fun to play through flow and hitsounding.
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/on-the-fm.jpg)](https://osu.ppy.sh/s/568544)
 
-This month we have something different in the chart, a gimmick map! Tired of maps with regular sv? Do you want to play maps with constant sv changes? On the FM it's your map! But hey, we are talking of the Inner Oni, right? No! You will have a sv challenge in all diffs, even in the Kantan. Nofool did a good choice with this song as the sv changes fits pretty well and gives a lot of variety, since 1/4 patterns to nonstop 1/3 patterns. What? Too slow? Use HR and you will enjoy more the map!
+This month we have something different in the chart, a gimmick map! Tired of maps with regular Slider Velocity? Do you want to play maps with constant speed changes? On the FM it's your map! But hey, we are talking of the Inner Oni, right? No! You will have a SV challenge in all diffs, even in the Kantan. **[Nofool](https://osu.ppy.sh/users/672430)** did a good choice with this song as the speed changes fits pretty well and gives a lot of variety, since 1/4 patterns to nonstop 1/3 patterns. What? Too slow? Use HardRock and you will enjoy more the map!
```

Comment:
I hope this comment looks better now x.x :

+This month we have something different in the chart, a gimmick map! Tired of maps with regular Slider Velocity? Do you want to play maps with constant speed changes? On the FM is your map! But hey, we are talking of the Inner Oni, right? No! You'll be challenged by the SV in all difficulties in this set, even the Kantan. **[Nofool](https://osu.ppy.sh/users/672430)** made a good choice with this song as the speed changes fit pretty well and adds a lot of variety, from 1/4 patterns to nonstop 1/3 patterns. What? Too slow? HardRock will add a bit of speed and spice up the set a little for you!